src/db.ts

```diff
@@ -226,35 +225,15 @@ export class Db {
    *
    * @param name - The name of the collection to create
    * @param options - Optional settings for the command
-   * @param callback - An optional callback, a Promise will be returned if none is provided
    */
-  createCollection<TSchema extends Document = Document>(
+  async createCollection<TSchema extends Document = Document>(
```

Comment:
I hope leaving a comment here will make github not collapse this file, easy to overlook

Reply:
it still collapses it, but that's fine if we use the github checkmarks for viewed files